### PR TITLE
Fix scaffold command after merge

### PR DESCRIPTION
**Pull request by amagdas**
TODO: add test

### CODE REVIEW

File: server/apps/prepopulate/app_scaffold_data.py [hunk 1]
```diff
@@ -56,7 +56,10 @@ def ingest_items_for(self, desk, no_of_stories, skip_index):
 
                 dest_doc[app.config['VERSION']] = 1
                 dest_doc['state'] = 'fetched'
-                send_to(dest_doc, desk_id, stage_id)
+                user_id = desk.get('members', [{'user': None}])[0].get('user')
+                dest_doc['original_creator']= user_id
+                dest_doc['version_creator']= user_id
```
Comment (plamut) [inline server/apps/prepopulate/app_scaffold_data.py]: Whitespace missing, flake8 will probably complain.

**Comment by ioanpocol** [comment]
What is the status of this PR?

**Comment by amagdas** [comment]
I still need to add some tests for this, will do as soon I have the time to do so.

**Comment by amagdas** [comment]
@superdesk/developers Review please

**Comment by marwoodandrew** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
:+1:

**Pull request merged**